docs/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md

```diff
@@ -5,13 +5,8 @@ description: "How to perform a backup and restore of Operate and Tasklist data."
 keywords: ["backup", "backups"]
 ---
 
-:::note
-This release introduces breaking changes, including:
-
-- The [get backup state API and response codes](#get-backup-state-api).
-- The utilized URL has changed. For example, `curl 'http://localhost:8080/actuator/backups'` rather than the previously used `backup`.
-- `backupId` must be of integer type now instead of string, which is in sync with Zeebe `backupId` requirements.
-
+:::warning breaking changes!
+In this release, the actuator endpoints (including backup) changed to use port 9600 instead of 8080. You will need to update your applications to use port 9600.
```

Comment:
I don't know what we define as a breaking change - this doesn't disrupt running applications/won't fail an upgrade, but may need a configuration adjustment.

```suggestion
As of the Camunda 8.6 release, the `/actuator` endpoints (including `/backups`) now default to port 9600. Ensure your `management.server.port` configuration parameter is correctly set before continuing.
```

Reply:
@ThorbenLindhauer @mihail-ca Does this definitely constitute a breaking change, or is it more of a configuration issue for people to be aware of?

Reply:
Yes, I'd say so. Default behavior of the product is changing, so applications that build on this default behavior will break. In this case, for example automations that people have written for orchestrating the backup procedures.